### ISSUE TITLE
Deduplicate - sum integers in string

### ISSUE THREAD

**Issue by hobovsky** [codewars/content-issues#143]
From #39

1. [Sum of integers in string](https://www.codewars.com/kata/598f76a44f613e0e0b000026)
* 7kyu
* Numbers are separated by letters and non-digits
* 92% with ~5k completions
* 2 pending issues, one about being a duplicate of 2.
* 12 languages + 4 pending translations
* Aug 2017, author inactive

2. [Sum up the random string](https://www.codewars.com/kata/55da6c52a94744b379000036)
* 7kyu
* Numbers are separated by letters and non-digits
* 93% with ~2.8k completions
* 1 pending issue
* 7 languages + 2 pending translations
* Aug 2015, author inactive

3. [Summy](https://www.codewars.com/kata/599c20626bd8795ce900001d)
* 7kyu
* Separator is a space
* 91% with 3.7k completions
* 1 pending issue
* 6 languages + 3 pending translations
* Aug 2017, author inactive

4. [How many animals are there?](https://www.codewars.com/kata/593406b8f3d071d83c00005d)
* 7kyu
* Numbers are separated by letters and non-digits
* 90% with ~1k completions
* No pending issues
* 2 languages, no pending translations
* Jun 2017, author last active Jun 2022

5. [Responsible Drinking](https://www.codewars.com/kata/5aee86c5783bb432cd000018)
* 7kyu
* Numbers are single digits only, and separated by letters and non-digits, output should be slightly formatted.
* 92% with ~5.5k completions
* 5 pending issues
* 7 languages + 1 pending translation
* May 2018, author active

6. [String Calculator](https://www.codewars.com/kata/55ffd58d186cc9394f0000cc)
* 7kyu
* Numbers are separated by `", "` (comma + space)
* 94% with ~670 completions
* No pending issues
* C# only
* Sep 2015, author inactive

### Languages

![image](https://user-images.githubusercontent.com/23709795/187427669-c3dfd922-ba39-4f24-9f7e-5c792959975f.png)

----

### Conclusion

- Keep 1. [Sum of integers in string](https://www.codewars.com/kata/598f76a44f613e0e0b000026)
- [x] Retire 2. [Sum up the random string](https://www.codewars.com/kata/55da6c52a94744b379000036) after CoffeeScript translation gets moved
- [x] Retire 3. [Summy](https://www.codewars.com/kata/599c20626bd8795ce900001d) after NIM translation gets moved
- [x] Retire 4. [How many animals are there?](https://www.codewars.com/kata/593406b8f3d071d83c00005d)
- Keep 5. [Responsible Drinking](https://www.codewars.com/kata/5aee86c5783bb432cd000018)
- [x] Retire 6. [String Calculator](https://www.codewars.com/kata/55ffd58d186cc9394f0000cc)

### Filling gaps

- [x] Move CoffeeScript translation from 2. to 1.
- [x] Move NIM translation from 3. to 1.

**Comment by hobovsky** [comment]
Kata 1., 2., and 4. seem to be exact duplicates and are slightly more demanding than the rest. I think 1. would be the best to keep due to amount of languages.
Kata 5. is similar to above, but numbers are always 1-9, and output must be formatted in a simple way.  Is it sufficiently different for a white kata? I think it's not, I'd vote to retire it.
Kata 3. and 6. are very similar to each other as they have fixed separators. Maybe 3. would be good to keep? I think 6. can go away.

**Comment by Blind4Basics** [comment]
keep 1, retire all others. They are all too much similar (when not nearly identical)

**Comment by Kacarott** [comment]
I think it is better to sort these kata by the solution required:
- Kata 1 & 2 require some form of parsing (probably regex) before summation
- Kata 3 & 6 merely require splitting on a fixed delimiter
- Kata 4 & 5 require splitting on a fixed delimiter, but requires an additional filter for non-numbers

Based on this, I think that the first group is sufficiently different from the rest, to warrant keeping one of them, and I would choose 1 based on number of translations. Between the rest, I think the additional filtering step is nice, especially at 7kyu. So between kata 4 & 5 I would choose 5 based once again on number of languages.

In summary: Keep 1 and 5, retire the rest.

**Comment by EloiseRosen** [comment]
keep 1 and 5 only sounds good to me

**Comment by KayleighWasTaken** [comment]
Keep 1, 3, 5 I think?

3 is the least attractive to keep, maybe rerank it as 8kyu if that's possible?

**Comment by hobovsky** [comment]
Added [Coffeescript translation](https://www.codewars.com/kumite/63d5ae1f0e8b7d13c91e9742?sel=63d5ae1f0e8b7d13c91e9742)
Added [Nim translation](https://www.codewars.com/kumite/63d5b6670e8b7d15d51e96ea?sel=63d5b6670e8b7d15d51e96ea)
